JS.md

```diff
@@ -205,6 +205,25 @@ to the Object `{ module: String(i.module_name), name: String(i.item_name), kind:
 
 Note: other fields like `signature` may be added in the future.
 
+### `WebAssembly.Module.userSections`
+
+The `userSections` function has the signature:
+
+```
+Array userSections(modueObject, sectionName)
+```
+
+If `moduleObject` is not a `WebAssembly.Module`, a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror)
+is thrown.
+
+Let `sectionNameString` be the result of [`ToString`](https://tc39.github.io/ecma262/#sec-tostring)(`sectionName`).
+
+This function returns an `Array` produced by mapping each
+[user-defined section](BinaryEncoding.md#high-level-structure) (i.e., section with
+`id` 0) whose `name` field ([decoded as UTF-8](Web.md#names)) is equal to
+`sectionNameString` to an `ArrayBuffer` containing the section's `payload_data`.
+(Note: `payload_data` does not include `name` or `name_len`.)
```

Comment:
I was wondering about this. Why make a copy? Don't we have the original buffer copied already (to prevent user from modifying it while compiling)? Maybe not a big deal, but it seems wasteful when we could just provide a view over the existing copy.